### ISSUE TITLE
Won't start on Debian GNU/Linux 7.10

### ISSUE THREAD

**Issue by DmitriSchrama** [nczempin/Turres-Monacorum#154]
I don't know if I should leave this here, but when I try to start the game in debian version 7.10 with amd64 architecture I get the following error:

 demian@Portable-Thinky:~/Downloads/nczempin-Turres-Monacorum-15fb0af$ love love2d/
conf.lua:6: attempt to index field 'window' (a nil value)
Error: external/postshader.lua:29: attempt to call field 'newShader' (a nil value)
stack traceback:
	external/postshader.lua:29: in main chunk
	[C]: in function 'require'
	main.lua:4: in main chunk
	[C]: in function 'require'
	[string "boot.lua"]:331: in function <[string "boot.lua"]:227>
	[C]: in function 'xpcall'

Hope this helps.

**Comment by nczempin** [comment]
it does help; what version of love2d would that be (Just in case you'd know this; I can research this myself)?

**Comment by Supersonic112** [comment]
If I were to take a guess: It's version 0.10.x, where love.window.getWidth() has been removed. The fastest and easiest way to circumvent this problem is probably to go back to love2d version 0.9.x.
I'll take a look at getting the game to run with love 0.10.1 in the next days if I have the time ;)

**Comment by nczempin** [comment]
That's nice; note that someone has done work on a fork of our project to get things working with a more current version of love2d; perhaps check that out and/or talk to the guy.

**Comment by DmitriSchrama** [comment]
It was indeed 0.8.0 that's the version in the repository. I'll get the one from the website and see what happens.

**Comment by nczempin** [comment]
We tested with löve 0.9.1, which was current when we wrote it, two years ago. I think there would be some issues with a more current version; as mentioned above I've seen someone do a fork and work on an update for a more recent version of Löve.

**Comment by nczempin** [comment]
https://gist.github.com/casaval/61b9808c90716ed6407f are steps to install 0.9.1 (the exact version we tested this with) in Debian Wheezy (7).

**Comment by nczempin** [comment]
Okay, and Debian Jessie (8) uses 0.9.1 as their stable version, it seems: https://packages.debian.org/jessie/games/love

**Comment by DmitriSchrama** [comment]
Right you are :) GNU/Linux Never forget RMS. :+1: 
So... I'm not that in to it. I usually just follow the instructions, but I don't feel like compiling a bunch of things to play a game. Tried to install older versions but I got some dependency errors. Seems I have to update my system. Then it will probably work.  So I'll come back when I've done that. Might take a while though. Good luck! And have fun! :)
